### DISCUSSION TITLE
Portfolio and Organizations

### DISCUSSION THREAD

**Discussion by cbupp** [RealEstateCore/rec#149] [Q&A] [answered]
Is there a relationship that shows that a collection (Portfolio, etc) and an organization are related.

In REC3.3, there was "Organization owns RealEstate" and "RealEstate isOwnedBy Organization"

I'm considering extending for my project to add, but don't want to diverge if there was something I'm missing:
Organization hasCollection Collection
Collection isCollectionOf Organization

**Comment by hammar** [comment]
Good catch. As of right now, there is no such explicit link between a Collection and an Agent. The idea has generally been that a Collection is an intangible/administrative grouping of things that themselves are "real"; e.g., a `Portfolio` of `Buildings` or other `Spaces`, an `Apartment` of `Rooms`, etc. In such a world view, it would be the physical real things that would be owned, rather than the collection. Similarly in this perspective, the collection probably should not have relations to, e.g., `Points`, or other non-administrative relationships.

However, we do have types that bridge this divide, where the `Collection` really is more tangible/real or ought to have relations on its own that are not purely administrative. E.g., the `System` as taken from Brick should probably have Points, or as you point out, the `RealEstate` type representing a building and/or site as a *legal entity* should probably have an owner. So, we may need to think on this and possibly revisit the design/criteria. 

One option to resolve your specific question might be to remove the `Architecture.ownedBy` relationship in favor of an inverse `Agent.owns` relationship with no target set, indicating that agents (natural persons or organizations/companies) can own all manners of things. 

I'll create an issue from this discussion point, and add it to the Technical Committee meeting agenda for October 3. Also pinging some colleagues in the TC on this thread: @PeteHart @erikoskarwallin @perkarlberg. Feel free to discuss solutions to this so that we can follow up on October 3 and ideally get a decision settled then.

Thanks for your contribution here and on the other issue @cbupp!

**Reply by cbupp** [reply]
@hammar - Any update from the October 3 meeting?

**Reply by hammar** [reply]
The meeting was delayed and is actually running just now. We've ended up in a big modelling principles discussion. We will need community feedback on this discussion, would love to hear your (and others) takes on it. 

I'll summarize here today or tomorrow, and ping on Gitter and other channels also.

**Comment by hammar** [comment]
Hey everyone pinged on this: as a result of the TC call yesterday, I've created a poll at #157. Please comment and vote there on how we should proceed in the general case of open-ended Relationship declarations; based on where we proceed from there, we can then more concretely resolve the Collection relationships discussed in this thread.

Sorry that it's taking a bit of time @cbupp, but we think it's important to get this one right.

**Comment by hammar** [comment, marked as answer]
Thanks to the community for all the discussions on this! We implemented Agent.owns in PR #191, so the ontology now has such a feature.
